docs/marketplace/dev/glue-api-guides/202200.0/shopping-lists/managing-shopping-lists.md

```diff
@@ -0,0 +1,1127 @@
+            "relationships": {
+                "concrete-products": {
+                    "data": [
+                        {
+                            "type": "concrete-products",
+                            "id": "005_30663301"
+                        }
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+</details>
+
+For the response attributes, see [Create a shopping list](#create-a-shopping-list).
+
+For the attributes of included resources, see:
+* [Add items to a shopping list](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-shopping-lists/managing-shopping-list-items.html#shopping-list-items-response-attributes)
+* [Retrieve a concrete product](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/concrete-products/retrieving-concrete-products.html#concrete-products-response-attributes)
+
+## Retrieve a shopping list
+
+To retrieve a shopping list, send the request:
+
+***
+`GET` **/shopping-lists/*{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}***
+***
+
+| PATH PARAMETER | DESCRIPTION |
+| --- | --- |
+| ***{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}*** | Unique identifier of a shopping list to retrieve. |
+
+
+### Request
+
+| HEADER KEY | TYPE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| Authorization | string | ✓ | String containing digits, letters, and symbols that authorize the company user. [Authenticate as a company user](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-b2b-account/authenticating-as-a-company-user.html#authenticate-as-a-company-user) to get the value.  |
+
+| QUERY PARAMETER | DESCRIPTION | EXEMPLARY VALUES |
+| --- | --- | --- |
+| include | Adds resource relationships to the request. | shopping-list-items, concrete-products, merchants, product-offers |
+
+{% info_block infoBox "Included resources" %}
+
+To retrieve concrete products in a shopping list, include `shopping-list-items` and `concrete-products` resources.
+
+{% endinfo_block %}
+
+| REQUEST SAMPLE | USAGE |
+| --- | --- |
+| `GET http://glue.mysprykershop.com/shopping-lists/sdb17f85-953f-565a-a4ce-e5cb02405f83` | Retrieve the shopping list with the id `sdb17f85-953f-565a-a4ce-e5cb02405f83`. |
+| `GET http://glue.mysprykershop.com/shopping-lists/ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a?include=shopping-list-items` | Retrieve the shopping list with the id `ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a` with its items. |
+| `GET http://glue.mysprykershop.com/shopping-lists/ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a?include=shopping-list-items,concrete-products` | Retrieve the shopping list with the id `ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a` with its items and respective concrete products. |
+| `GET http://glue.mysprykershop.com/shopping-lists/c0bc6296-8a0c-50d9-b25e-5bface7671ce?include=shopping-list-items` | Retrieve the shopping list with the id `c0bc6296-8a0c-50d9-b25e-5bface7671ce` with its items (marketplace products and product offers). |
```

Comment:
This is not the request URL that returns product offers & merchant products.

Reply:
removed the example